pandas/tests/io/test_excel.py

```diff
@@ -363,12 +363,12 @@ def test_reader_converters(self):
 
         basename = 'test_converters'
 
-        expected = DataFrame.from_items([
```

Comment:
feel free to change some / most constructions with from items entirely (rather than catch the deprecation warning)